src/ringcentral-web-phone.js

```diff
@@ -356,8 +381,8 @@
         session.on('cancel', stopPlaying);
         session.on('failed', stopPlaying);
         session.on('replaced', stopPlaying);
-        session.mediaHandler.on('iceConnectionCompleted', stopPlaying);
-        session.mediaHandler.on('iceConnectionFailed', stopPlaying);
+        // session.sessionDescriptionHandler.on('iceConnectionCompleted', stopPlaying);
+        // session.sessionDescriptionHandler.on('iceConnectionFailed', stopPlaying);
```

Comment:
Remove if it's not needed, don't leave it commented

Reply:
Removed

Reply:
I have asked to remove! Not uncomment!